auth/auth_test.go

```diff
@@ -193,6 +192,60 @@ func TestCustomTokenInvalidCredential(t *testing.T) {
 	}
 }
 
+func TestVerifyIDTokenWithCheckRevokedValid(t *testing.T) {
+	s := echoServer(testGetUserResponse, t)
+	defer s.Close()
+
+	ft, err := s.Client.VerifyIDTokenWithCheckRevoked(ctx, testIDToken, true)
```

Comment:
Add a test case where validSince is in the past.

Reply:
This test has validSince before the token 'iat'
TestVerifyIDTokenWithCheckRevokedInvalidated puts the token 'iat' before the valid since, is that what you meant?